source/scone/input/keyboard_event.d

```diff
@@ -6,30 +6,28 @@ import scone.core.flags : hasFlag;
 
 struct KeyboardEvent
 {
-    this(SK key, SCK controlKey, bool pressed)
-    {
-        this.key = key;
-        this.controlKey = controlKey;
-        this.pressed = pressed;
-    }
-
     auto hasControlKey(SCK ck)
     {
         return controlKey.hasFlag(ck);
     }
 
     public SK key;
     public SCK controlKey;
-    public bool pressed;
+    version (Windows) public bool pressed = true;
 }
 
 unittest
 {
-    assert(KeyboardEvent(SK.a, SCK.none, true).key == SK.a);
-    assert(KeyboardEvent(SK.a, SCK.ctrl, true).controlKey == SCK.ctrl);
-    assert(KeyboardEvent(SK.a, SCK.none, false).pressed == false);
+    assert(KeyboardEvent(SK.a, SCK.none).key == SK.a);
+    assert(KeyboardEvent(SK.a, SCK.ctrl).controlKey == SCK.ctrl);
+
+    version (Windows)
+    {
+        assert(KeyboardEvent(SK.a, SCK.none, false).pressed == false);
+        assert(KeyboardEvent(SK.a, SCK.none, true).pressed == true);
+    }
 
-    auto keyboardEvent = KeyboardEvent(SK.a, SCK.ctrl | SCK.alt, true);
+    auto keyboardEvent = KeyboardEvent(SK.a, SCK.ctrl | SCK.alt);
```

Comment:
(Example of why KeyboardEvent.pressed defaults to true. This is usually the most common case)